report_thesis/src/sections/background/linear_and_regularization_models/ols.tex

```diff
@@ -0,0 +1,12 @@
+\subsubsection{Ordinary Least Squares Regression}\label{sec:ols}
+We briefly cover the \gls{olsr} algorithm based on \citet{James2023AnIS} to provide context for the subsequent discussions on linear and regularization models.
+
+\gls{olsr} is a linear regression technique that fits a linear model, $\hat{y} = \hat{\beta}_0 + \hat{\beta}_1 x$, to the data by minimizing the residual sum of squares, where $\hat{y}$ is the predicted target value, $\hat{\beta}_0$ is the intercept, $\hat{\beta}_1$ is the coefficient for the input feature, and $x$ is the input feature.
+It does this by estimating the coefficients that minimize the objective function, specifically the sum of squared differences between the observed target values and the predicted values, known as the \gls{rss}:
+
+$$
+\text{RSS} = \sum_{i=1}^{n} (y_i - \hat{y}_i)^2,
+$$
+
+where $y_i$ is the observed target value, $\hat{y}_i$ is the predicted target value, and $n$ is the number of samples in the dataset.
+The objective of \gls{olsr} is to find the coefficients $\hat{\beta}_0$ and $\hat{\beta}_1$ that minimize the \gls{rss}.
```

Comment:
This has already been said in line 5, however this relates more to the RSS equation. Perhaps it makes sense to combine it all in one place - Either before or after the RSS definition.